src/sysusers/sysusers.c

```diff
@@ -64,13 +64,19 @@ typedef struct Item {
         uid_t uid;
 
         bool gid_set:1;
-        bool gid_must_exist:1;
+        bool gid_must_exist_and_can_be_reused:1;
         bool uid_set:1;
 
         bool todo_user:1;
         bool todo_group:1;
 } Item;
 
+typedef enum uid_is_ok_flags {
+        DEFAULT = 0,
+        SKIP_GID_REUSE_CHECK = 1,
```

Comment:
First of all, I think it is not necessary to add this. Of course, in general, more flags may be used in the future. But I think we should add such enum type when we need, not now.
Second, if you want to use these flags, please rename this e.g. `UID_IS_OK_SKIP_GID` or something. I am afraid that I cannot catch the meaning of `REUSE_CHECK`. Also, please remove `DEFAULT`. I guess when we really need this enum, then each flag will be bit field, right?

Reply:
If you think a bool is enough, thats fine with me, I updated the PR.